tests/Integration/PrestaShopBundle/Form/FormClonerTest.php

```diff
@@ -50,7 +50,7 @@ class FormClonerTest extends AbstractFormTester
     public function testClone(): void
     {
         $form = $this->createForm(AdvancedFormType::class);
-        $this->assertTotalNumberOfListeners($form, 5);
+        $this->assertTotalNumberOfListeners($form, 6);
```

Comment:
in form listeners (f`orm.pre_submit`, `form.post_submit`, `form.post_set_data`) `DataCollectorListener` which is now in `post_submit`, the number of listeners has therefore been incremented